var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -81,15 +79,11 @@ def flag_handler(self, name, flags):
                 # The following flag forces the compiler to produce module
                 # files with lowercase names.
                 flags.append('-ef')
-        elif name == 'ldflags':
-            # We need to specify LDFLAGS to get correct dependency_libs
-            # in libnetcdff.la, so packages that use libtool for linking
-            # could correctly link to all the dependencies even when the
-            # building takes place outside of Spack environment, i.e.
-            # without Spack's compiler wrappers.
-            config_flags = [self.spec['netcdf-c'].libs.search_flags]
```

Comment:
From my naive perspective, this looks problematic: this was removed but nothing appears to have been added to take its place (I see added logic in `netcdf-cxx4`'s `flag_handler` method which appears to do something similar to this, but I assume this would need it too). Presumably you removed this because it causes an error on Mac OS, but is it still needed on Linux?

Reply:
@scheibelp as you can see in the removed comment, this was only needed to keep the libtool's `.la` file consistent. But Spack removes such files for quite some time already (see #18850). As for the linking itself, Spack's compiler wrapper adds the required `-L` flags anyway.